pom.xml

```diff
@@ -127,6 +128,7 @@
         <retrofit.version>2.5.0</retrofit.version>
         <saaj.impl.version>1.5.3</saaj.impl.version>
         <scala-2.12.version>2.12.15</scala-2.12.version><!-- Spark -->
+        <scala.version>2.13.3</scala.version><!-- Datasonnet -->
```

Comment:
Scala version 2.12 is currently dictated by Quarkus. We could override it in our BOM, but any application generated by code.quarkus.io would get the version from Quarkus BOM anyway. 
Hence we have only two options here: (a) ask Quarkus to upgrade to Scala 2.13 or (b) make datasonnet work with 2.12.
I see this as the main issue of this PR.

Reply:
Sorry, I don't think it's possible to make Datasonnet work with 2.12...so I guess we're in the holding pattern until Quarkus upgrades to 2.13

Reply:
OK, thanks for confirming that it cannot work with Scala 2.12. There is this quite old quest to upgrade in Quarkus: https://github.com/quarkusio/quarkus/issues/10025 I have added a comment there.

Reply:
Scala 2.13 in Quarkus is underway https://github.com/quarkusio/quarkus/pull/25106